src/main/kotlin/com/noahbres/meepmeep/MeepMeep.kt

```diff
@@ -319,6 +319,18 @@ open class MeepMeep @JvmOverloads constructor(private val windowSize: Int, fps:
                 colorManager.isDarkMode = false
                 rotated(ImageIO.read(classLoader.getResourceAsStream("background/season-2023-centerstage/field-2023-juice-light.png")), Math.toRadians(90.0))
             }
+            Background.FIELD_INTOTHEDEEP_OFFICIAL -> {
```

Comment:
I prefer `INTO_THE_DEEP` for easy reading and to match the service mark (as appears to have been done in the past)